LiteCore/Query/SQLiteFleeceFunctions.cc

```diff
@@ -103,6 +107,77 @@ namespace litecore {
         }
     }
 
+    static void handle_fts_value(const Value* data, alloc_slice& result) {
+        switch(data->type()) {
+            case kBoolean:
+                result.append(data->asBool() ? "T"_sl : "F"_sl);
+                break;
+            case kNumber:
+            {
```

Comment:
For the entire `kNumber` case, you can call `toString()` to convert it to string form.